packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -80,10 +87,14 @@ export class MeasureToolsUiItemsProvider implements UiItemsProvider {
 
       if (tools.length > 0 && toolbarOrientation === ToolbarOrientation.Horizontal) {
         const isHidden = new ConditionalBooleanValue(
-          () => !MeasurementUIEvents.isClearMeasurementButtonVisible,
+          () => {
+            const vp = IModelApp.viewManager.selectedView;
+            return (vp ? !MeasurementUIEvents.isClearMeasurementButtonVisible || ViewHelper.isSheetView(vp) : !MeasurementUIEvents.isClearMeasurementButtonVisible);
```

Comment:
why not
!MeasurementUIEvents.isClearMeasurementButtonVisible && ViewHelper.isSheetView(vp)